proto/aquila/action_execute.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+option ruby_package = "Tucana::Aquila";
+
+package aquila;
+
+message ActionCallRequest {
+  repeated string parameters = 1;
+}
+
+message ActionCallResponse {
+  string execution_identifier = 1;
+}
+
+message ActionResult {
+  repeated string result = 1;
+}
+
+message ActionResultRequest {}
+
+message ActionResultResponse {
+  string execution_identifier = 1;
```

Comment:
```suggestion
```

You can simply remove this `execution_identifier` because the execution already has the id and send's it over via the request as the linked identifier